### PR TITLE
chore: add installation and setup docs.

### PR DESCRIPTION
**Pull request by i4ki**

### CODE REVIEW

File: README.md
Comment (katcipis) [inline README.md]:
```suggestion
The only `terramate-ls` specific setup required is making sure it is installed in a
```

**Review by katcipis — COMMENTED** [review]
LGTM + small possible typo

**Review by katcipis — DISMISSED** [review]

**Comment by i4ki** [comment]
waiting for the project's name change to merge this.

**Comment by i4ki** [comment]
The docs mention installing using release binaries but there's no release yet.
I'm going to make the very first release after merging this.

File: README.md
Comment (soerenmartius) [inline README.md:11]:
```suggestion
To install using Go run:
```

**Review by soerenmartius — APPROVED** [review]
lgtm, minor suggestion

**Review by katcipis — APPROVED** [review]

**Pull request merged**
